aws-rds-dbcluster/src/main/java/software/amazon/rds/dbcluster/BaseHandlerStd.java

```diff
@@ -102,6 +104,9 @@
 
     protected HandlerConfig config;
 
+    private final Collection<String> SENSITIVE_PARAMETERS_PARENTS = ImmutableSet.of("desiredResourceState", "previousResourceState", "resourceModel");
```

Comment:
I believe we should aim for a more generic approach tbh, a hardcoded structure could be a limiting factor for the solution applicability in the future.